### DISCUSSION TITLE
Releasing GAMA 1.8.2 soon ?

### DISCUSSION THREAD

**Discussion by RoiArthurB** [gama-platform/gama.old#3079] [Q&A]
The development of GAMA 1.8.2 is nearly finished : 
- Only [2 issues](https://github.com/gama-platform/gama/issues?q=is%3Aissue+is%3Aopen+label%3A%22Version+1.8.2%22) left
- Running fine with the latest 15 JDK
- Compatible with MacOS Big Sure (main point of this issue)
- The latest alpha release (from 03/03/2021) have [more than 40 downloads](https://benoitgaudou.github.io/testGitHubAPI/) without new issues

I think we can release that new version! :)

**Comment by AlexisDrogoul** [comment]
A few more tests need to be done, though, on two completely different aspects: 
- GIS related operations, as GeoTools as has been moved to v. 25.0
- light/dark themes, which do not seem to work very well in all environments.

Finally, I would be happy to receive a -- hopefully positive -- report about 1.8.2 working on Apple M1 Computers

**Comment by RoiArthurB** [comment]
About the Light/Dark themes, I have an excellent default one on Linux (it's the same theme as Dark Eclipse) 
![image](https://user-images.githubusercontent.com/16764085/113530328-ee242280-95ef-11eb-92ce-2d33021a52cf.png)

**Reply by ndgnuh** [reply]
The source code is not readable though.

**Reply by AlexisDrogoul** [reply]
That's also my point of view... :)
